src/hotspot/share/runtime/threads.cpp

```diff
@@ -754,9 +744,8 @@ jint Threads::create_vm(JavaVMInitArgs* args, bool* canTryAgain) {
 #endif
 
 #if INCLUDE_JVMCI
-  if (force_JVMCI_intialization) {
+  if (force_JVMCI_initialization) {
     JVMCI::initialize_compiler(CHECK_JNI_ERR);
```

Comment:
Does anything bad happen if force_JVMCI_initialization is true and we end up initializing the compiler from the compile broker thread instead?  I guess that would be adding Xcomp to the the other flags that force initialization.

Reply:
Ok.